airflow/www/yarn-error.log

```diff
@@ -0,0 +1,8373 @@
+Arguments: 
+  /usr/bin/node /usr/share/yarn/bin/yarn.js install --frozen-lockfile
+
+PATH: 
+  /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+Yarn version: 
+  1.22.5
+
+Node version: 
+  10.23.1
+
+Platform: 
```

Comment:
Is this file supposed to be a part fo this PR?

Reply:
Nope.  Good catch.  Ty.